lib/jekyll-github-metadata/repository.rb

```diff
@@ -64,6 +64,10 @@ def releases_url
         "#{repository_url}/releases"
       end
 
+      def latest_release_url
+        "#{releases_url}/latest"
```

Comment:
Thank you for the clarification! Just updated to use the `latest_release` object (hash).